controllers/suite_test.go

```diff
@@ -125,6 +135,7 @@ var _ = AfterSuite(func() {
 	// restore Manifestpaths for both controller to their original value
 	ManifestPath = "./bindata/deployment"
```

Comment:
General: you use this path multiple times in the code can you add them as consts?